CRM/Contribute/Form/Search.php

```diff
@@ -153,8 +153,8 @@ public function preProcess() {
    */
   public function setDefaultValues() {
     $this->_defaults = parent::setDefaultValues();
-    if (empty($this->_defaults['contribution_status'])) {
-      $this->_defaults['contribution_status'][1] = 1;
+    if (empty($this->_defaults['contribution_status_id'])) {
+      $this->_defaults['contribution_status_id'][1] = 1;
```

Comment:
Hardcoded?
```
$this->_defaults['contribution_status_id'][1] = CRM_Core_PseudoConstant::getKey(
  'CRM_Contribute_BAO_Contribution',
  'contribution_status_id',
  'Completed'
);
```

Reply:
Thanks @pradpnayak good point have changed to the PseudoConstant